neuralop/training/tests/test_callbacks.py

```diff
@@ -194,4 +196,69 @@ def test_load_from_checkpoint():
 
     # clean up dummy checkpoint directory after testing
     shutil.rmtree('./full_states')
+    
+    
+# enure that the model incrementally increases in frequency modes
+def test_incremental():
+    # Loading the Darcy flow dataset
+    train_loader, test_loaders, output_encoder = load_darcy_flow_small(
```

Comment:
nit: thinking about test runtime. Might there be a way to verify that the behavior is as expected without loading such a big dataset?